src/crimson/os/cyanstore/cyan_store.cc

```diff
@@ -405,7 +405,7 @@ seastar::future<> CyanStore::do_transaction(CollectionRef ch,
       {
         coll_t cid = i.get_cid(op->cid);
         ghobject_t oid = i.get_oid(op->oid);
-        std::map<std::string, ceph::bufferlist> aset;
+        std::map<std::string, ceph::bufferptr> aset;
```

Comment:
the comparator was left out intentionally? (here and below)

Reply:
@ronen-fr i am still working on the crimson side.